tpu_inference/kernels/fused_moe/v1/kernel.py

```diff
@@ -19,8 +19,7 @@ def align_to(x, a):
 
 
 def get_dtype_packing(dtype):
-    bits = (dtypes.bit_width(dtype)
-            if hasattr(dtypes, "bit_width") else dtypes.itemsize_bits(dtype))
+    bits = dtypes.itemsize_bits(dtype)
```

Comment:
can you revert this change? this is for backward compatibility with older jax version. note that we are currently working on defaulting to 0.8.1 and we can remove this if/else once that lands.